kubernetes/resource_kubernetes_service.go

```diff
@@ -198,6 +235,34 @@ func resourceKubernetesServiceSchemaV1() map[string]*schema.Schema {
 							string(api.ServiceAffinityNone),
 						}, false),
 					},
+					"session_affinity_config": {
+						Type:        schema.TypeList,
+						Description: "Contains the configurations of session affinity. More info: https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-ipvs",
+						Optional:    true,
+						Computed:    true,
+						MaxItems:    1,
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								"client_ip": {
+									Type:        schema.TypeList,
+									Description: "",
+									Optional:    true,
+									Computed:    true,
+									MaxItems:    1,
+									Elem: &schema.Resource{
+										Schema: map[string]*schema.Schema{
+											"timeout_seconds": {
+												Type:        schema.TypeInt,
+												Description: "",
```

Comment:
There is a description for this field: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#clientipconfig-v1-core

Looks like we might be able to add some validation here too, or at least validate that it's non-negative.

Reply:
Good catch, I have added descriptions and the following validation `validation.IntBetween(1, 86400)`.